src/services/functionAppService.test.ts

```diff
@@ -222,6 +226,37 @@ describe("Function App Service", () => {
     expect(uploadCall[2]).toMatch(/.*-t([0-9]+)/)
   });
 
+  it("uploads functions to function app and blob storage with default naming convention", async () => {
+    const scmDomain = app.enabledHostNames.find((hostname) => hostname.endsWith("scm.azurewebsites.net"));
+    const expectedUploadUrl = `https://${scmDomain}/api/zipdeploy/`;
+
+    const sls = MockFactory.createTestServerless();
+    delete sls.service["artifact"]
+    const service = createService(sls);
+    await service.uploadFunctions(app);
+
+    const defaultArtifact = path.join(".serverless", `${sls.service.getServiceName()}.zip`);
+
+    expect((FunctionAppService.prototype as any).sendFile).toBeCalledWith({
```

Comment:
nit: I would extract the `toBeCalledWith` arguments out just so it's easier to read